discourse-doctor.rb

```diff
@@ -116,7 +124,16 @@ def check_plugins
   end
 end
 
+def check_available_disk_space
+  stat = Filesystem.stat(DISCOURSE_PATH.to_s)
+  bytes_available = stat.blocks_available * stat.block_size
+  if bytes_available < 10 * 1024
+    warning("The available disk space at #{DISCOURSE_PATH} is less than 10MB")
```

Comment:
@ZogStriP this check is before setup right ? So what would be a good value once Discourse is setup?